apps/portainer-ce/2.24.1-alpine/docker-compose.yml

```diff
@@ -10,7 +10,7 @@ services:
     volumes:
       - ./data:/data
       - /var/run/docker.sock:/var/run/docker.sock
-    image: portainer/portainer-ce:2.24.1-alpine
+    image: portainer/portainer-ce:2.25.0-alpine
     labels:
       createdBy: "Apps"
 networks:
```

Comment:
There is only one change in the `image` field from "portainer/portainer-ce:2.24.1-alpine" to "portainer/portainer-ce:2.25.0-alpine". This is a standard version update without major changes that generally won't introduce significant issues or require optimizations.